### DISCUSSION TITLE
WordPress Editor style does not pick up on custom font or fallback

### DISCUSSION THREAD

**Discussion by filip-van-hoeckel** [tailpress/tailpress#214] [Q&A]
I managed to get custom fonts working using the discussions here, However my editor styles apparently do not pick up the correct values and render all editor content in Times/serif. How and where do I start troubleshooting this behaviour?

**Comment by charlie-george-1989** [comment]
@filip-van-hoeckel Did you manage to resolve this? If not, give me a shout.

**Comment by filip-van-hoeckel** [comment]
Nope, it doesn't hurt performance but it does make us look bad with the client.

**Reply by charlie-george-1989** [reply]
Okay, well let me do some testing and I'll get back to you when I find a solution.

**Reply by imhugos** [reply]
Hi! This is happening to me also. Any news on this?

**Reply by filip-van-hoeckel** [reply]
We still experience this on the latest version of tailpress. FWIW we use Google fonts using Bunny, as per documentation.
